data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg

```diff
@@ -125,7 +125,9 @@
         side=1
         controller=human
         recruit=Thug,Poacher,Spearman,Bowman,Footpad
-        gold=0 #player has large carryover here, is handled in prestart event
+        # player is supposed to have a large carryover here (handled in prestart event), but just in case:
+        {GOLD 20 10 0}
```

Comment:
Have you seen a bug here? The code in the prestart seems fine, hence the question about whether there's anything being fixed here.

20 gold seems insignificant - IIRC that "large carryover" is expected to be in the 5000-8000 range.

Reply:
ditto what @stevecotton said: this value is effectively only a placeholder anyway, since prestart event will overwrite it. Differentiating it per difficulty in this place makes no sense.

Reply:
I think the change here was to help sniff out if I was in fact seeing a bug with the prestart event or not...

Reply:
I put it back to 0 in [d11e74e](https://github.com/wesnoth/wesnoth/pull/6057/commits/d11e74ebe7abe3f47e5391b5b24d31152db65866).